docs/src/main/sphinx/functions/string.rst

```diff
@@ -137,16 +150,16 @@ String functions
 .. function:: soundex(char) -> string
 
    ``soundex`` returns a character string containing the phonetic representation of ``char``.
-    It is typically used to evaluate the similarity of two expressions phonetically, that is 
+    It is typically used to evaluate the similarity of two expressions phonetically, that is
     how the string sounds when spoken::
 
-        SELECT name 
+        SELECT name
         FROM nation
         WHERE SOUNDEX(name)  = SOUNDEX('CHYNA');
 
          name  |
         -------+----
-         CHINA | 
+         CHINA |
```

Comment:
Unrelated changes.

Reply:
Sure. They just remove whitespace so it wont matter .. do you still want @findinpath to revert those?

Reply:
I'll just add these changes in a separate commit to keep the context of documenting `luhn_check` clear of unrelated changes. My IDE did these changes automatically and unfortunately I didn't notice them.